k8s/gcp/k8s-installer/create-k8s-cluster.yml

```diff
@@ -19,15 +19,22 @@
       shell: python ../../utils/name_generator/namesgenerator.py
       register: cluster_name
       when: not cluster_name
+
+    - name: Get VPC Name
+      shell: cat ~/logs/vpc
+      register: vpc
+
     - set_fact:
         cluster_name: "{{ cluster_name.stdout }}"
-      when: cluster_name.stdout is defined
+    - set_fact:
+        vpc: "{{ vpc.stdout }}"
+        
     - name: Creating Bucket
       shell: gsutil mb gs://{{ cluster_name }}/
     - name: Creating the Cluster & InstanceGroup objects in our state store
       shell: |
         export KOPS_FEATURE_FLAGS=AlphaAllowGCE
-        kops create cluster {{ cluster_name }}.k8s.local --kubernetes-version {{ k8s_version }} --zones us-east1-b --state gs://{{ cluster_name }}/ --project {{ project }} --node-count {{ nodes }} --networking kubenet --image "ubuntu-os-cloud/ubuntu-1604-xenial-v20170202" --vpc {{ vpc_name }}
+        kops create cluster {{ cluster_name }}.k8s.local --kubernetes-version {{ k8s_version }} --zones us-east1-b --state gs://{{ cluster_name }}/ --project {{ project }} --node-count {{ nodes }} --networking kubenet --image "ubuntu-os-cloud/ubuntu-1604-xenial-v20170202" --vpc {{ vpc }}
     - name: Creating K8s Cluster
       shell: |
         export KOPS_FEATURE_FLAGS=AlphaAllowGCE
```

Comment:
Can we validate if the cluster gets created successfully.

Reply:
Yes, for this we are having a Python script under `utils` that checks for the cluster availability!